packages/features/form-builder/FormBuilder.tsx

```diff
@@ -483,16 +488,36 @@ function FieldEditDialog({
                       }
                       label={t("identifier")}
                     />
-                    <InputField
-                      {...fieldForm.register("label")}
-                      // System fields have a defaultLabel, so there a label is not required
-                      required={
-                        !["system", "system-but-optional"].includes(fieldForm.getValues("editable") || "")
-                      }
-                      placeholder={t(fieldForm.getValues("defaultLabel") || "")}
-                      containerClassName="mt-6"
-                      label={t("label")}
-                    />
+                    <div>
+                      {fieldForm.getValues("type") === "boolean" ? (
+                        <div className="mt-6">
+                          <Label>{t("label")}</Label>
+                          <Editor
+                            getText={() => md.render(fieldForm.getValues("label") || "")}
+                            setText={(value: string) => {
+                              fieldForm.setValue("label", turndown(value), { shouldDirty: true });
```

Comment:
Can we abstract this component out in a CheckboxField or something similar component

Reply:
Done